modules/nw-egress-ips-about.adoc

```diff
@@ -88,6 +88,11 @@ The {rh-openstack} egress IP address feature creates a Neutron reservation port
 ====
 endif::openshift-sdn[]
 
+[NOTE]
+====
+When a cluster administrator assigns a floating IP to the reservation port, {product-title} cannot delete the reservation port. The `CloudPrivateIPConfig` object cannot perform delete and move operations until the floating IP is unassigned from the reservation port.
```

Comment:
Can we make it more specific that this is about OpenStack cluster administrator, not OpenShift? Right now this is ambiguous.

I think we can use `{rh-openstack}` to achieve that.

Reply:
@dulek Addressed.

Reply:
Does the note look good now?

Reply:
Looks good from my perspective.